WcaOnRails/app/assets/stylesheets/competitions.scss

```diff
@@ -169,7 +169,11 @@ $venue-map-wrapper-height: 400px;
 
   #state {
     min-width: 220px;
-    width: 100%;
+    display: block;
+    @media (max-width: $screen-sm) {
```

Comment:
This looks like something special for mobile, can you include a screenshot on a tiny and large screen?